code/unused/goonheist/genetics2/geneticsMachines.dm

```diff
@@ -1,974 +0,0 @@
-var/list/genetics_computers = list()
```

Comment:
Keep the goonheist folder. This is goonetics, and should be preserved for historical purposes, even if it isn't being used.